openassessment/xblock/grade_mixin.py

```diff
@@ -49,8 +49,11 @@ def render_grade(self, data, suffix=''):
         context = {}
 
         # Render the grading section based on the status of the workflow
+        # but give precedence to showing the override score if it is set.
         try:
-            if status == "done":
+            if workflow.get('override_score', None):
```

Comment:
What are we actually checking here? that:
- workflow has been assigned some value for the key `override_score`
- or workflow has been assigned some value for the key `override_score` _and_ its value is Truthy

Assuming the former is what we really want, we should change this to simply check for the key's existence:
```python
if 'override_score' in workflow:
    # ...
```